File: dyros_robot_menagerie/dyros_robot_menagerie/husky/robot_data.py
from drc import KinematicParam, DriveType
from drc.mobile.robot_data import RobotData

"""
URDF Joint Information: Husky
 name                | value
---------------------+---------------------------
type                 | Differential
wheel_num            | 2
wheel_radius         | 0.1651
base_width           | 1.0702
"""
TASK_DOF  = 3
WHEEL_DOF = 2

class HuskyRobotData(RobotData):
    def __init__(self):
        param = KinematicParam(type         = DriveType.Differential,
                               wheel_radius = 0.1651,
                               base_width   = 0.2854 * 2 * 1.875,
                               max_lin_acc  = 3,
                               max_ang_acc  = 6,
                               )
        super().__init__(param)

File: dyros_robot_menagerie/src/husky/robot_data.cpp
#include "dyros_robot_menagerie/husky/robot_data.h"

namespace Husky
{
    static drc::Mobile::KinematicParam makeParam()
    {
        drc::Mobile::KinematicParam p;
        p.type         = drc::Mobile::DriveType::Differential;
        p.wheel_radius = 0.1651;
        p.base_width   = 0.2854 * 2 * 1.875;
        p.max_lin_acc  = 3;
        p.max_ang_acc  = 6;
        return p;
    }

    HuskyRobotData::HuskyRobotData()
    : drc::Mobile::RobotData(makeParam())
    {

    }
} // namespace Husky